### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cwpeng**
Update Project to version 1.0.1

### CODE REVIEW

**Review by docats — CHANGES_REQUESTED** [review]
bad bad

**Review by ba40431 — CHANGES_REQUESTED** [review]
No

**Review by f789520 — CHANGES_REQUESTED** [review]
太帥

**Comment by itdog1973** [comment]
no good re do junior

**Review by cony1220 — APPROVED** [review]

**Review by CurtisCY — APPROVED** [review]
OK

**Comment by pingdori** [comment]
讚

**Review by alilerlian — APPROVED** [review]
good!

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-這是 GA 範例
\ No newline at end of file
+這是 GA 範例，即將廢棄
```
Comment (peichang6) [inline README.md:1]: 這裡棒棒

**Review by Lishainyun — APPROVED** [review]
gogo

**Review by joesound — APPROVED** [review]
GOOD

**Review by pingdori — APPROVED** [review]
OK

**Review by jenniehuang — CHANGES_REQUESTED** [review]
夭壽贊

**Review by ba40431 — APPROVED** [review]
100分

**Review by klina916 — APPROVED** [review]
very good good

**Review by joesound — APPROVED** [review]
good pass

**Review by jenniehuang — COMMENTED** [review]
彭彭好帥

**Review by itdog1973 — APPROVED** [review]
寫太好啦 過！

**Review by cony1220 — APPROVED** [review]
讚

**Review by pingdori — APPROVED** [review]
讚讚讚

**Review by f789520 — APPROVED** [review]
帥

**Comment by alilerlian** [comment]
yes

**Review by CurtisCY — APPROVED** [review]

**Review by avivi28 — APPROVED** [review]
棒~

File: README.md (hunk 1 above)
Comment (docats) [reply README.md]: 讚讚讚

**Review by alilerlian — APPROVED** [review]
good again

**Review by peichang6 — APPROVED** [review]

**Review by docats — COMMENTED** [review]
good

**Pull request merged**